<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="up">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Down">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>170</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Door_OBS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Door Obstacles</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fire">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>fire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>370</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="power_outage">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>power outage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overload">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>370</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Overloaded </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="floor_lvl">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>210</y>
      <width>45</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Output">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>351</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Solved">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>SOLVED</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>190</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Floor</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Total_Elevators">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="Total_floors">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevators</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>24</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Floors </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Custom">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Custom amount</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter custom amount of elevators here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DoorClose">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Door Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DoorOpen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>DoorOpen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
